example/lib/main.dart

```diff
@@ -95,6 +95,28 @@ class _MyAppState extends State<MyApp> {
     }
   }
   
+  void writeFileOnButtonPress() async {
+    var creds = ClientCredentials(apiKey: "", 
+                                  apiSecret: "",
+                                  clientId: "",
+                                  publicKey: "",
+                                  privateKey: "",
+                                  publicSignKey: "",
+                                  privateSigningKey: "",
+                                  host: "",
+                                  email: "",
+                                  clientName: "");
+    var client = PluginTozny(creds);
+    try {
+      RecordMeta writtenFileMeta = await writeFile(client);
```

Comment:
I feel like I am missing something. Does this need more parameters about record type and the file path to upload?

Reply:
writeFile is already a method within main.dart (https://github.com/tozny/flutter_plugin/blob/wash-932-write-file-record/example/lib/main.dart#L202) & the record type & file path are defined there.